### ISSUE TITLE
Mobilitydojo database error

### ISSUE THREAD

**Issue by qubex22** [ahelland/EASMD#2]
Hi, first of all, great job documenting everything about AS. Your posts from mobilitydojo are being very helpful to me. 

However there are some posts that are not available, I know they are very old, but I was wondering if you'd have a backup or something. The posts are the ones pointed here: [http://mobilitydojo.net/2011/08/10/exchange-activesync-building-blocks-intro/](http://mobilitydojo.net/2011/08/10/exchange-activesync-building-blocks-intro)

The error I get when I try to access the page is: _Error establishing a database connection_

Hope you can help me, thanks for your work 👍

**Comment by ahelland** [comment]
Yeah, known issue. Had a look at it a while back without quite understanding why. (Apart from it being related to the database of course.)
I have easy access to things like the files in those posts if that's what you're looking for. The actual posts - less sure how to extract those.

**Comment by qubex22** [comment]
I am interested in the explanation, because I am trying to understand the full Activesync Protocol and your posts there are the only ones I have found, that cover everything in a painless way (not like microsoft documentation). Particularly I am interested in the provision stage. I succesfully provisioned a device applying the directives, but now I am trying to reproduce a provision without applying them (marking on the server the option to allow unprovisioned devices), and I can't get it working

Anyways, don't worry too much, I will continue my research. Thanks!!
